hack/setup-test-cluster.sh

```diff
@@ -25,30 +31,36 @@ do
     ${OC_TOOL} label $node ptp/slave=""
 done
 
-# Note: this is intended to be the only pool we apply all mcs to.
-# Additional mcs must be added to this poll in the selector
-cat <<EOF | ${OC_TOOL} apply -f -
-apiVersion: machineconfiguration.openshift.io/v1
-kind: MachineConfigPool
-metadata:
-  name: worker-cnf
-  labels:
-    machineconfiguration.openshift.io/role: worker-cnf
-spec:
-  machineConfigSelector:
-    matchExpressions:
-      - {
-          key: machineconfiguration.openshift.io/role,
-          operator: In,
-          values: [worker-cnf, worker],
-        }
-  maxUnavailable: null
-  paused: false
-  nodeSelector:
-    matchLabels:
-      node-role.kubernetes.io/worker-cnf: ""
----
+apply_machine_config_pool() {
+    # Additional mcs must be added to this poll in the selector
+    cat <<EOF | ${OC_TOOL} apply -f -
+    apiVersion: machineconfiguration.openshift.io/v1
+    kind: MachineConfigPool
+    metadata:
+      name: ${CNF_LABEL}
+      labels:
+        machineconfiguration.openshift.io/role: ${CNF_LABEL}
+    spec:
+      machineConfigSelector:
+        matchExpressions:
+          - {
+              key: machineconfiguration.openshift.io/role,
+              operator: In,
+              values: [${CNF_LABEL}, worker],
+            }
+      maxUnavailable: null
```

Comment:
This should be removed. https://github.com/openshift-kni/cnf-features-deploy/pull/107